_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -88,8 +88,107 @@ Option | Required | Type | Description
 :--- | :--- | :--- | :---
 `start_position` | No | String | The position from where the source starts reading stream events when the DynamoDB stream option is enabled. `LATEST` starts reading events from the most recent stream record. 
 
+## Exposed metadata attributes
+
+The following metadata will be added to each event that is processed by the `dynamodb` source. These metadata attributes can be accessed with the [expression syntax `getMetadata` function](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/#getmetadata).
+
+* `primary_key` - The primary key of the DynamoDB item. For tables that only contain a partition key, this value gives the partition key. For tables that contain both a partition and sort key, the` primary_key` attribute will be equal to the partition and sort key, separated by a `|`, for example `partition_key|sort_key`.
+* `partition_key` - The partition key of the DynamoDB item.
+* `sort_key` - The sort key of the DynamoDB item. This will be null if the table does not have a sort key.
+* `dynamodb_timestamp` - The timestamp of the DynamoDB item. This will be the export time for export items and the DynamoDB stream event time for stream items. This timestamp is used by sinks to emit an `EndtoEndLatency` metric for DynamoDB stream events, which tracks the latency between a change in the DynamoDB table and that change getting applied to the sink.
+* `document_version` - Uses the `dynamodb_timestamp` to modify break ties between stream items that are received in the same second. Recommend for use with the `opensearch` sink's `document_version` setting.
+* `opensearch_action` - A default value for mapping DynamoDB event actions to OpenSearch actions. This action will be `index` for export items, and `INSERT` or `MODIFY` for stream events, and `REMOVE` stream events when the OpenSearch action is `delete`.
```

Comment:
"This action will be `index` for export items, `INSERT` or `MODIFY` for stream events, and `REMOVE` when the OpenSearch action is `delete`"? (The second sentence needs a little revision for consistency.)